ArmPlatformPkg/Drivers/PL061GpioDxe/PL061Gpio.c

```diff
@@ -242,14 +242,14 @@ Set (
 
     case GPIO_MODE_OUTPUT_0:
       // Set the corresponding direction bit to HIGH for output
-      MmioOr8 (RegisterBase + PL061_GPIO_DIR_REG, GPIO_PIN_MASK (Offset));
+      MmioOr8 (RegisterBase + PL061_GPIO_DIR_REG, (UINT8)GPIO_PIN_MASK (Offset));

```

Comment:
Instead a of the repeatedly casting maybe change GPIO_PIN_MASK to be UINT8 in the header file. Please see the comment on PL061Gpio.h

Reply:
done